editor.planx.uk/src/@planx/components/MapAndLabel/Public/index.tsx

```diff
@@ -94,44 +104,60 @@ const VerticalFeatureTabs: React.FC<{ features: Feature[] }> = ({
                   {`${feature.geometry.type}`}
                   {feature.geometry.type === "Point"
                     ? ` (${feature.geometry.coordinates.map((coord) =>
-                      coord.toFixed(5),
-                    )})`
-                    : ` (area ${feature.properties?.["area.squareMetres"] || 0
-                    } m²)`}
+                        coord.toFixed(5),
+                      )})`
+                    : ` (area ${
+                        feature.properties?.["area.squareMetres"] || 0
+                      } m²)`}
                 </Typography>
               </Box>
-              <Box>
-                <InputLabel label="Copy from" id={`select-${i}`}>
-                  <SelectInput
-                    bordered
-                    required
-                    title={"Copy from"}
-                    labelId={`select-label-${i}`}
-                    value={""}
-                    onChange={() => console.log(`TODO - Copy data from another tab`)}
-                    name={""}
-                    style={{ width: "200px" }}
-                  >
-                    {features.filter((feature) => feature.properties?.label !== activeTab).map((option) => (
-                      <MenuItem
-                        key={option.properties?.label}
-                        value={option.properties?.label}
-                      >
-                        {`${schema.type} ${option.properties?.label}`}
-                      </MenuItem>
-                    ))}
-                  </SelectInput>
-                </InputLabel>
-              </Box>
+              {features.length > 1 && (
```

Comment:
Only show the copy select once we have multiple features.

Reply:
a11y question: would it be better to be always visible _but disabled_ until we have multiple features? Maybe with conditional `visuallyHidden` text that announces something like "Add another feature first before copying" 

Just thinking if we conditionally render we'll have to probably sort out aria announcements which might get complicated/awkward quickly - eg:
- Plot a point, add tab one which is active tab by default, no copy
- Plot another point, add tab two which is now active tab, has copy - announce that tab one (the inactive tab) also has copy available now??

Reply:
This is a very good point which I've overlooked, I'll pick this up in https://github.com/theopensystemslab/planx-new/pull/3594